.storybook/main.js

```diff
@@ -8,11 +8,11 @@ module.exports = {
   ],
   addons: [
     "@storybook/addon-a11y",
-    "@storybook/addon-essentials",
     "@storybook/addon-links",
     "@storybook/addon-storysource",
     "@storybook/preset-create-react-app",
-    "storybook-addon-performance/register"
+    "storybook-addon-performance/register",
+    "@storybook/addon-essentials",
```

Comment:
Per storybook docs, for the actions addon (bundled with addon-essentials) to be guaranteed to work with other addons it should be the last addon loaded to take higher precedence